tools/matRad_generateBodyContour.m

```diff
@@ -0,0 +1,59 @@
+function cst = matRad_generateBodyContour(ct,cst,thresholdHU)
+% function to create a BODY contour for imported patient cases that do not have one
+% 
+% call
+%   cst = matRad_generateBodyContour(ct,cst,thresholdHU)
+%
+% input
+%   ct:             matrad ct structure
+%   cst:            matrad cst structure
+%   thresholdHU:    HU thresholding value (optional) default = -500 HU
+%                  
+%
+% output
+%   cst:        matRads cst struct  with inserted BODY contour
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+% Copyright 2018 the matRad development team.
+%
+% This file is part of the matRad project. It is subject to the license
+% terms in the LICENSE file found in the top-level directory of this
+% distribution and at https://github.com/e0404/matRad/LICENSES.txt. No part
+% of the matRad project, including this file, may be copied, modified,
+% propagated, or distributed except according to the terms contained in the
+% LICENSE file.
+%
+% %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+if nargin<3
+    thresholdHU = -500;
```

Comment:
Isn't this dangerously high in case of Lung patients?
Or is this handled by imfill?

Reply:
Also just realized that imfill is a image processing toolbox function, is there an easy alternative? Otherwise make a check for the toolbox and make sure it also works in Octave by using the same mechanism to load Octave's image package that we use in other places (e.g. the dicom import).

Reply:
yup supposed to be handled by the imfill, but i noticed that for a H&N CT, you could have the trachea as an open ended tube which leads the contour in as well, but for our current purposes i think it works.